tests/execution_engine/test_sqlalchemy_batch_data.py

```diff
@@ -172,8 +169,8 @@ def test_instantiation_with_temp_table_schema():
         create_temp_table=True,
         temp_table_schema_name="test_schema",
     )
-    query_to_create_temp_table: str = batch_data._create_temporary_table(
-        temp_table_name="hello",
+    (query_to_create_temp_table, temp_table_name) = batch_data._create_temporary_table(
+        dialect=GXSqlDialect.SQLITE,
```

Comment:
Test had to be adjusted because the underlying method changed, but the behavior is still the same :)